geometry_msgs/Pose pose
int32 num_nodes_left